dist/docker/entrypoint.sh

```diff
@@ -1,21 +1,27 @@
 #!/bin/sh
 
-if [ ! -f /config/qBittorrent/qBittorrent.conf ]; then
-    mkdir -p /config/qBittorrent/
-    cat << EOF > /config/qBittorrent/qBittorrent.conf
+profilePath="/config"
+qbtConfigFile="$profilePath/qBittorrent/config/qBittorrent.conf"
+
+if [ ! -f "$qbtConfigFile" ]; then
+    mkdir -p "$(dirname $qbtConfigFile)"
+    cat << EOF > "$qbtConfigFile"
 [BitTorrent]
 Session\DefaultSavePath=/downloads
+Session\Port=6881
 Session\TempPath=/downloads/temp
 
 [LegalNotice]
 Accepted=false
 EOF
-fi
 
-if [ "$LEGAL" = "accept" ] ; then
-    sed -i '/^\[LegalNotice\]$/{$!{N;s|\(\[LegalNotice\]\nAccepted=\).*|\1true|}}' /config/qBittorrent/qBittorrent.conf
-else
-    sed -i '/^\[LegalNotice\]$/{$!{N;s|\(\[LegalNotice\]\nAccepted=\).*|\1false|}}' /config/qBittorrent/qBittorrent.conf
+    if [ "$QBT_EULA" = "accept" ]; then
+        sed -i '/^\[LegalNotice\]$/{$!{N;s|\(\[LegalNotice\]\nAccepted=\).*|\1true|}}' "$qbtConfigFile"
+    else
+        sed -i '/^\[LegalNotice\]$/{$!{N;s|\(\[LegalNotice\]\nAccepted=\).*|\1false|}}' "$qbtConfigFile"
+    fi
 fi
 
-HOME="/config" XDG_CONFIG_HOME="/config" XDG_DATA_HOME="/data" qbittorrent-nox --webui-port=$WEBUI_PORT
+exec qbittorrent-nox \
+    --profile="$profilePath" \
```

Comment:
What do the contents of the `/config` dir look like now- are there subfolders for `/config` and `/data`, or are they combined?

Reply:
This is the file tree:
```
config/
└── qBittorrent
    ├── cache
    ├── config
    │   ├── categories.json
    │   ├── lockfile
    │   ├── qBittorrent.conf
    │   ├── rss
    │   │   ├── feeds.json
    │   │   └── storage.lock
    │   └── watched_folders.json
    └── data
        ├── BT_backup
        │   └── queue
        ├── GeoDB
        │   └── dbip-country-lite.mmdb
        ├── logs
        │   └── qbittorrent.log
        └── rss
            └── articles
                └── storage.lock
```